### PR TITLE
add playing sound feature

### PR DESCRIPTION
**Pull request by get-me-power**
I add playing sound effect function if call  sl#animate().

Please check this code.

### CODE REVIEW

**Comment by kazukazuinaina** [comment]
I found bug. I close this pull request.
Sorry.

File: autoload/sl.vim [hunk 1]
```diff
@@ -63,6 +63,7 @@ let s:data = [
 \]
 
 function! sl#animate()
+  let filepath = expand('**/sl.wav')
```
Comment (mattn) [inline autoload/sl.vim]: remove this line.

File: autoload/sl.vim [hunk 2]
```diff
@@ -78,6 +79,7 @@ function! sl#animate()
   let l:wrap_backup = &l:wrap
   setlocal nowrap
   redraw
+  call sound_playfile(filepath)
```
Comment (mattn) [inline autoload/sl.vim]:
```vim
if exists('*sound_playfile')
  let l:filepath = expand('<sfile>:h:h') . '/resource/sl.wav'
  call sound_playfile(l:filepath)
endif
```

File: README.md [hunk 3]
```diff
@@ -24,3 +24,7 @@ MIT
 ## Author
 
 Yasuhiro Matsumoto (a.k.a. mattn)
+
+## Sound
+
+aonrjp
```
Comment (mattn) [inline README.md]: Please add the link to download original file.

File: autoload/sl.vim (hunk 1 above)
Comment (get-me-power) [reply autoload/sl.vim]: Thank you! I will fix.

File: README.md (hunk 3 above)
Comment (get-me-power) [reply README.md]: OK.

File: autoload/sl.vim [hunk 4]
```diff
@@ -79,7 +78,9 @@ function! sl#animate()
   let l:wrap_backup = &l:wrap
   setlocal nowrap
   redraw
-  call sound_playfile(filepath)
+  if exists('*sound_playfile')
+    let l:filepath = expand('**/sl.wav')
+    call sound_playfile(filepath)
```
Comment (mattn) [inline autoload/sl.vim:83]: endif

Comment (get-me-power) [reply autoload/sl.vim:83]: Sorry...

File: README.md (hunk 3 above)
Comment (get-me-power) [reply README.md]: Prease wait...

I will add URL soon.

File: autoload/sl.vim (hunk 4 above)
Comment (mattn) [reply autoload/sl.vim:83]: No prob :smile:

Comment (get-me-power) [reply autoload/sl.vim:83]: Thank you very much.
I fix some problem.
Please check this code.

File: README.md (hunk 3 above)
Comment (mattn) [reply README.md]: Thanks for the link to wav file. What the license of the file? Whose is this?

Comment (get-me-power) [reply README.md]: Thanks to reply.

"aonr.jp" create this wav file. (github url : https://github.com/greenlaver)

This sound's license is public domain.

File: autoload/sl.vim (hunk 4 above)
Comment (mattn) [reply autoload/sl.vim:83]: Thanks. So you should move up sl.wav from autoload/sl.wav to base directory?

File: README.md (hunk 3 above)
Comment (mattn) [reply README.md]: Thanks. Then, please update README.md like below.
```
## Thanks

sl.wav : https://aonr.jp/
```

File: autoload/sl.vim (hunk 4 above)
Comment (get-me-power) [reply autoload/sl.vim:83]: OK. I will transfer this wav file to base directory.

File: README.md (hunk 3 above)
Comment (get-me-power) [reply README.md]: I fixed. Please check!

File: autoload/sl.vim (hunk 4 above)
Comment (get-me-power) [reply autoload/sl.vim:83]: I fixed. Please check!

File: autoload/sl.vim [hunk 5]
```diff
@@ -78,6 +78,10 @@ function! sl#animate()
   let l:wrap_backup = &l:wrap
   setlocal nowrap
   redraw
+  if exists('*sound_playfile')
+    let l:filepath = expand('<sfile>:h:h') . '/sl.wav'
+    call sound_playfile(filepath)
```
Comment (mattn) [inline autoload/sl.vim:83]: Sorry, nit pick. `l:filepath`

Comment (get-me-power) [reply autoload/sl.vim:83]: And I'm sorry many times to take a mistake...

**Comment by mattn** [comment]
Merged. https://github.com/mattn/vim-sl/commits/master

Thank you.

**Pull request closed without merging**